.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -269,6 +252,53 @@ as well as an upgrade of the previous version of Teleport.
     - [ ] `tsh ssh -Y server01 "echo Hello World | xclip -sel c && xclip -sel c -o"` should print "Hello World"
     - [ ] `tsh ssh -X server01 "echo Hello World | xclip -sel c && xclip -sel c -o"` should fail with "BadAccess" X error
 
+### Web UI integrations
+
+NOTE: Web UI integrations should be tested on an Enterprise Cloud tenant.
+
+Instructions for deploying a custom release to a cloud staging tenant: https://github.com/gravitational/teleport.e/blob/master/dev-deploy.md
+
+NOTE: The integration enrollment wizards will have you run scripts in AWS cloudshell
+that try to download a teleport tarball from our usual CDN. That won't work with a pre-release build.
+You will need to modify the downloaded scripts to download the pre-release
+tarball from `cdn.cloud.gravitational.io` instead of `cdn.teleport.dev`, e.g:
+  `bash -c $(curl <url> | sed 's/cdn.teleport.dev/cdn.gravitational.io/')`
+
+- [ ] External Audit Storage
+
+  - [ ] Discover flow works to configure External Audit Storage https://goteleport.com/docs/choose-an-edition/teleport-cloud/external-audit-storage/
+    - [ ] Docs (including screenshots) are up to date
+    - [ ] Discover flow works with or without an existing AWS OIDC integration
+    - [ ] Draft configuration can be resumed after navigating away
+    - [ ] Bootstrap step (oneoff command pasted into CloudShell) works to create infra
+    - [ ] Created IAM policy (attached to AWS OIDC integration role) matches docs example
+    - [ ] Audit Events and Session Recordings (created after EAS enabled) are stored in configured S3 buckets
+    - [ ] Audit Events and Session Recordings (created after EAS enabled) can be queried and played in the web UI
+    - [ ] `tsh play <session-id>` works
+  - [ ] Existing EAS configuration can be replaced with a new one via Discover flow
+  - [ ] Existing EAS configuration can be deleted (disabling EAS)
+
+- [ ] RDS Enrollment
```

Comment:
I found it in the other test plan template:
https://github.com/gravitational/teleport/blob/master/.github/ISSUE_TEMPLATE/webtestplan.md#enroll-new-resources-using-discover-wizard

Reply:
I moved the EAS stuff to the web test plan and added my notes there as well

Reply:
https://github.com/gravitational/teleport/compare/002e763ef0c6ed8ae9a32377a097c1c392215b3e..fa859053599524a016acbd95a9b4dd482f679e5a